docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -123,6 +123,46 @@ Use the terminal for the following steps:
 
    You can also use ``conda info --envs``.
 
+.. _specifying-environment-platform:
+
+Specifying a different target platform for an environment
+=========================================================
+
+By default, ``conda`` will create environments targeting the platform
+currently running on. You can check which one is yours by running
+``conda info`` and checking the ``platform`` entry.
+
+However, in some cases you might want to create an environment for a
+different target platform or architecture. To do so, use the
+``--platform`` flag available in the ``conda create`` and
+``conda env create`` commands. The value must be a valid platform.
+
+For example, a user running macOS on the Apple Silicon platform
+might want to create a ``python`` environment for Intel processors
+and emulate the executables with Rosetta. The command would be:
+
+.. code::
+
+   conda create --platform osx-64 --name python-x64 python
+
+Note that you can't specify this flag for existing environments.
+The environment will be annotated with the custom configuration and
+subsequent operations on it will remember the target platform.
+
+This flag also allows specifying a different OS (e.g. create a Linux
+environment on macOS), but we don't recommend its usage outside
+``--dry-run`` operations. Common problems include:
```

Comment:
```suggestion
``--dry-run`` operations. Common problems with mismatched OSes include:
```

I think it sounds better to reiterate the circumstance (mismatched OSes) under which these problems occur.